.rubocop.yml

```diff
@@ -342,6 +342,10 @@ Naming/AccessorMethodName:
   StyleGuide: '#accessor_mutator_method_names'
   Enabled: false
 
+Naming/InclusiveLanguage:
+  Description: 'Recommend the use of inclusive language instead of problematic terms.'
+  Enabled: false
```

Comment:
This is disabled because it targets all kinds of files, it'd be better to be enabled and configured separately and look over every violation